src/components/Results/ResultsRow.tsx

```diff
@@ -40,48 +38,63 @@ export const ResultsRow: FC<{result: Result}> = ({result}) => {
   const {solutions, rank_changed, rank_start, registration, subtotal, total} = result
 
   return (
-    <div className={styles.rowWrapper}>
-      <div className={styles.rank}>
+    // `display: 'contents'` je taka HTML verzia fragmentu - tento div sa v strukture nepocita pre ucely parent gridu,
+    // miesto toho sa pouzivaju jednotlive children. preto aj hover tu targetuje children divy, lebo parent v podstate nema velkost
+    // takisto `justifyContent: 'center'` je na vsetkych elementoch osobitne
+    <Box sx={{display: 'contents', ':hover > div': {bgcolor: 'black', color: 'white'}}}>
+      <Stack
+        sx={{
+          justifyContent: 'center',
+          // top-level PageLayout padding je vypnuty, odsadzame to rucne tu
+          pl: 1,
+        }}
+      >
         <Typography variant="h1" component="span">
           {rank_changed && rank_start + '.'}
         </Typography>
-      </div>
-      <div className={styles.nameAndSchool}>
-        <Typography variant="h2" component="span" className={styles.name}>
+      </Stack>
+      <Stack sx={{px: '10px', justifyContent: 'center'}}>
+        <Typography variant="h3" component="span">
```

Comment:
prislo ti h2 proste velke? alebo na zaklade coho si to zmenil na h3?

Reply:
noo, ze v mobil dizajne uz to bolo navrhnute mensie... a tym padom som to bral ako akceptaciu, ze pred tym (na desktope) su tie velkosti prilis v nesulade 😄 radsej by som vychadzal z tejto verzie, ked to este chceme menit na nejaku specialnu font variantu, ako z toho pred tym 😄

Reply:
dik za review, a teda necham tak, beriem, ze je to pripadne stale in progress/zive, ako toto bude finalne vyzerat..

Reply:
rozmyslal som si ze ci si sa inspiroval telefonovym font size :-D ale mi to prislo zaroven ze nee, ved si robil nieco ine

Reply:
jo, nechal som sa uniest, ked na mobile to bolo too much 😀